### ISSUE TITLE
Error on building example projects

### ISSUE THREAD

**Issue by andr-ew** [grrrwaaa/maxcpp#18]
Getting this error when I'm trying to build any of the example projects unchanged:

**The i386 architecture is deprecated. You should update your ARCHS build setting to remove the i386 architecture. (in target 'max-external')**

macOS 10.15.5, Max 8.0.1, Xcode 10.2.1

About to test, but I might guess it's the XCode version

Thank you!

**Comment by andr-ew** [comment]
well I fixed the issue for me, but might be helpful to know:

removed `i386` from the `ARCHS` setting in `maxmspsdk.xcconfig`

that fixed the above issue, then I got this error:

**/Users/instrument/Documents/Max 8/Packages/max-sdk-8.0.3/source/maxcpp-master/maxcpp/maxcpp6.h:51:10: 'new' file not found**

I followed the instructions [here](https://forums.developer.apple.com/thread/106114), changing "Apple Clang - Language -  C++ / C++ Standard Library" in Build Settings from  "libstdc++ (GNU C++ standard library)" to "libc++ (LLVM C++ standard library with C++11 support)"

Yea, probably just Xcode 10 stuff, but might be good to know

**Comment by grrrwaaa** [comment]
Yes this will be the fact that apple deprecated i386 architecture in recent versions of Xcode. It's an awkward situation as Max still runs in 32-bit mode on many systems, thus i386 compilation is still required for a time at least. Using an older version of Xcode can work, or if you don't want to care about 32-bit support, you can just upgrade the Xcode project as recommended to remove i386 arch support.
